stories/Components/Navigationcomponents/Mainnavigation/GlobalHeader/GlobalHeader.jsx

```diff
@@ -1,5 +1,6 @@
 import React, { useEffect } from 'react';
 import pnud from '../../../../assets/images/undp-logo-blue.svg';
+import { getData } from '../../../../assets/js/navigation-data';
```

Comment:
@Ashish-srij Change the function name to `getMegaMenu`.

Reply:
@shishirphadke Done.